src/site/content/en/blog/sms-otp-form/index.md

```diff
@@ -0,0 +1,266 @@
+not the most secure method of authentication by itself because phone
+numbers can be recycled and sometimes hijacked. And [the concept of OTP
+itself is not phishing resistant](https://youtu.be/kGGMgEfSzMw?t=1133).
+
+If you are looking for better security, consider using
+[WebAuthn](https://www.w3.org/TR/webauthn-2/). Learn more about it from the talk
+"[What's new in sign-up & sign-in](https://goo.gle/webauthn-video)" at the
+Chrome Dev Summit 2019. {% endAside %}
+
+## Checklist
+
+To provide the best user experience with the SMS OTP, follow these steps:
+
+* Use the `<input>` element with:
+    * `type="text"`
+    * `inputmode="numeric"`
+    * `autocomplete="one-time-code"`
+* Use `@BOUND_DOMAIN #OTP_CODE` as the last line of the OTP SMS message.
+* Use the [Web OTP API](/web-otp/).
+
+## Use the `<input>` element
+
+Using a form with an `<input>` element is the most important best practice you
+can follow because it works in all browsers. Even if other suggestions from
+this post don't work in some browser, the user will still be able to enter and submit the OTP 
+manually.
+
+```html
+<form action="/verify-otp" method="POST">
+  <input type="text"
+         inputmode="numeric"
+         autocomplete="one-time-code"
+         pattern="\d{6}"
+         required>
+</form>
+```
+
+The following are a few ideas to ensure an input field gets the best out of
+browser functionality.
+
+{% Aside %}
+For more general form best practices, [Sam Dutton](/authors/samdutton/)'s
+[Sign-in form best practices](/sign-in-form-best-practices/) is a great starting
+point.
+{% endAside %}
+
+### `type="text"`
+
+Because OTPs are usually five or six digit numbers, using
+`type="number"` for an input field might seem intuitive because it changes the mobile 
+keyboard to numbers only. This is not recommended because the browser expects an
+input field to be a countable number rather than a sequence of multiple numbers,
+which can cause unexpected behavior. Using `type="number"` causes up and down
+buttons to be displayed beside the input field; pressing these buttons
+increments or decrements the number and may remove preceding zeros.
+
+Use `type="text"` instead. This won't turn the mobile keyboard into numbers
+only, but that is fine because the next tip for using `inputmode="numeric"` does
+that job.
+
+### `inputmode="numeric"`
+
+Use `inputmode="numeric"` to change the mobile keyboard to numbers only.
+
+Some websites use `type="tel"` for OTP input fields since it also 
+turns the mobile keyboard to numbers only (including `*` and `#`) when
+focused. This hack was used in the past when
+[`inputmode="numeric"`](https://developer.mozilla.org/docs/Web/HTML/Global_attributes/inputmode)
+wasn't widely supported. Since [Firefox started supporting
+`inputmode="numeric"`](https://github.com/mdn/browser-compat-data/pull/6782),
+there's no need to use the semantically incorrect `type="tel"` hack.
```

Comment:
```suggestion
Use [`inputmode="numeric"`](https://developer.mozilla.org/docs/Web/HTML/Global_attributes/inputmode) to change the mobile keyboard to numbers only.

Some websites use `type="tel"` for OTP input fields since it also 
turns the mobile keyboard to numbers only (including `*` and `#`) when
focused. This hack was used in the past when `inputmode="numeric"`
wasn't widely supported. Since [Firefox started supporting
`inputmode="numeric"`](https://github.com/mdn/browser-compat-data/pull/6782),
there's no need to use the semantically incorrect `type="tel"` hack.
```